src/modules/MinifyImage/Module.js

```diff
@@ -76,7 +78,7 @@ module.exports = function MinifyImage(options, UI) {
         });
         var destPath = __dirname + '/results/test.' + input.format;
         var data = base64Img.base64Sync(destPath);
-        output(data, input.format);
+        output(destPath, data, input.format);
```

Comment:
Why do we do this in the browser? Why is it even required? Can't we just output the image? Was it used for debugging and was left out?

Reply:
This is used for command line interface.

Reply:
But that part is handled by the sequencer core library code anyway.